frontend/src/components/WorkoutPlan/RecordExercise.tsx

```diff
@@ -124,7 +124,25 @@ const RecordExercise: FC<RecordExerciseProps> = ({ route, navigation }) => {
     <>
       {isSetUploading && <Loader variant="Screen" />}
       <View style={[layout.sizeFull, layout.flex1, spacing.pdBottomBar]}>
-        <WorkoutVideoPopup width={width} videoId={extractVideoId(exercise.linkToVideo || "")} />
+        {exercise.linkToVideo ? (
+          <WorkoutVideoPopup width={width} videoId={extractVideoId(exercise.linkToVideo || "")} />
+        ) : (
+          <View
+            style={[
+              { width: width, height: 200 },
+              colors.backgroundSecondaryContainer,
+              layout.center,
+              spacing.gapDefault,
+            ]}
+          >
+            <NativeIcon
+              library="AntDesign"
+              name="exclamationcircleo"
+              style={[fonts.xxxxl, colors.textOnSecondaryContainer]}
+            />
+            <Text style={[colors.textOnSecondaryContainer]}>סרטון לא נמצא</Text>
+          </View>
+        )}
```

Comment:
dont use ? : (short-if) statements to return JSX its always messy and hard to read. 

Instead use && and check both conditions